src/test/java/com/tshprecher/postgres/antlr4/CommandTest.java

```diff
@@ -1,75 +1,89 @@
 package com.tshprecher.postgres.antlr4;
 
-import org.antlr.v4.runtime.*;
-import org.junit.Assert;
-import java.io.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-public abstract class CommandTest {
-
-    public void test(String commandName, String testDir) throws IOException {
-        if (testDir == null) {
-            printSummary(commandName, 0, 0, 0);
-            Assert.fail("test dir cannot be empty");
-            return;
-        }
-
-        String markerPath = getClass().getResource(testDir).getPath();
-        File dir = new File(markerPath.substring(0, markerPath.lastIndexOf('/')));
-        File[] testFiles = dir.listFiles();
-        int falsePositive=0, falseNegative=0, fileCount=0;
+import org.antlr.v4.runtime.BaseErrorListener;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.RecognitionException;
+import org.antlr.v4.runtime.Recognizer;
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
 
-        final AtomicBoolean success = new AtomicBoolean(true);
-        for (File f : testFiles) {
-            String name = f.getPath().substring(f.getPath().lastIndexOf("/"));
-            try (FileInputStream fio = new FileInputStream(f)) {
-                fileCount++;
-                StringBuilder header = new StringBuilder();
-                char ch;
-                while ((ch = (char)fio.read()) != '\n') {
-                    header.append(ch);
-                }
+public abstract class CommandTest {
 
-                boolean expect = header.toString().indexOf("expect:true") > 0;
-                if (!expect && header.toString().indexOf("expect:false") == -1) {
-                    Assert.fail("unexpected header format: cannot read expectation of file " + name);
-                }
+	public void test(String commandName, String testDir) throws IOException {
```

Comment:
Please use four spaces instead of tabs. It makes the diff easier to read. Furthermore, I don't think this file needs much of a change except to change the target from `stmt` to `root`.